README.md

```diff
@@ -1 +1,177 @@
+
+**[Capstone 2]** is a project that uses a Single Page Application format and two APIs to display items on a page. A user can open comments and reservations popup windows. He can also add comments and reservations to the individual popup windows. There are also like buttons which can be used to increment the number of likes for a selected movie season.
+
+## 🛠 Built With <a name="built-with"></a>
+-HTML
+-CSS
+-JavaScript
+
+### Key Features <a name="key-features"></a>
+
+- **[The project was built using ES6]**
+- **[Modules were imported/exported for the JavaScript]**
+- **[Webpack bundling was used]**
+- **[TVMaze and Invovlement APIs were consumed]**
+- **[GitHub Projects was used to track progress throughout the app building]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Project Presentation <a name="getting-started"></a>
+
+Click on the link below for the project presentation:
+
+- [Presentation Link](https://drive.google.com/file/d/1EzH2a3eMOoDAoMNG2LJM0yU8dbKfCCcN/view?usp=sharing)
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://uthmanbello.github.io/capstone2/dist)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-a code editor installed
+-linters environment fully setup
+### Setup
+
+Clone this repository to your desired folder:
+
+To clone this repository to your desired folder, use your code editor terminal or git bash, then type the command below:
+
+git clone https://github.com/Uthmanbello/capstone2
+
+### Install
+
+1. Install WebHint:
+npm install --save-dev hint@7.x
+
+2. Install StyleLint
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project, execute the following command:
+
+- open cloned folder in your code editor
+
+### Run tests
+
+To run tests, run the following command:
+
+To run tests and check for errors:
+- After installing the required linter,
+npx stylelint "**/*.{css,scss}"
+
+- To fix CSS or SCSS linters error:
+npx stylelint "**/*.{css,scss}" --fix
+
+-To fix JS linters errors
+npx eslint . --fix
+
+### Deployment
+
+You can deploy this project using:
+
+-[Deployment Link](https://uthmanbello.github.io/capstone2/dist)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author 1**
+
+- GitHub: [Uthmanbello](https://github.com/Uthmanbello)
+- Twitter: [@UthmanDeRoyale](https://twitter.com/UthmanDeRoyale)
+- LinkedIn: [Uthman Igein Bello-Imoukhuede](linkedin.com/in/uthman-igein-bello-imoukhuede)
+
+👤 **Author 2**
+
+- GitHub: [kkolade](https://github.com/kkolade)
+- Twitter: [@kola_kolade](https://twitter.com/kola_kolade)
+- LinkedIn: [Kola Kolade](https://linkedin.com/in/kolakolade/)
+
+👤 **Author 3**
```

Comment:
-  [ ] OPTIONAL
Generally, the subheading text _Author1_ and _Author2_ or similar should be replaced with your respective full names in the ReadMe.